firmware/quadruna/VC/src/tasks.c

```diff
@@ -310,6 +313,9 @@ void tasks_init(void)
         Error_Handler();
     }
 
+    io_imu_init();
```

Comment:
do something with the return?

Reply:
Logging a warning over CAN is probably the best bet